### PR TITLE
bonnie-prior 1302 nokogiri/loofah/rails-html-sanitizer fixes

### PR DESCRIPTION
**Pull request by c-monkey**
This PR fixes vulnerabilities found running `bundle exec bundle-audit check --update`. It also adds `bundle-audit` to travis to identify vulnerabilities that arise in future PRs.

Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [X] This pull request describes why these changes were made.
- [X] This PR is into the correct branch.
- [X] Internal ticket for this PR: https://jira.mitre.org/browse/BONNIE-1302
- [x] JIRA ticket links to this PR
- [X] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
- [X] Tests are included and test edge cases
- [X] Tests have been run locally and pass (remember to update Gemfile when applicable)
- [X] Code coverage has not gone down and all code touched or added is covered.
    * In rare situations, this may not be possible or applicable to a PR. In those situations:
        1. Note why this could not be done or is not applicable here:
        2. Add TODOs in the code noting that it requires a test
        3. Add a JIRA task to add the test and link it here:
- [X] Automated regression test(s) pass

**Reviewer 1:**

Name: @losborne
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

**Reviewer 2:**

Name: @hossenlopp
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -12,17 +17,21 @@ GEM
     libv8 (3.16.14.11)
```
Comment (lukoktonos) [inline Gemfile.lock]: `libv8 3.16.14.11` does not install for me when I run `bundle install` unless I do `gem install libv8 -v '3.16.14.11' -- --with-system-v8`.  Should we update this to match master, which has `3.16.14.19`?

Comment (c-monkey) [reply Gemfile.lock]: I'm glad you caught this, as it wasn't an error on my machine. bundler updating conservatively gives the 3.16.14.19 version, so I'm going to go ahead and do that.

**Review by lukoktonos — APPROVED** [review]
Looks good.  bundle install, audit, and tests all working.

File: .travis.yml [hunk 2]
```diff
@@ -1,7 +1,9 @@
 language: ruby
 rvm:
   - 1.9.3
```
Comment (hossenlopp) [inline .travis.yml]: Perhaps this should be updated to 2.3.5 to align with bonnie.

Comment (hossenlopp) [reply .travis.yml]: 2.3.4 to match bonnie-prior.

Comment (c-monkey) [reply .travis.yml]: sounds good to me!

**Review by hossenlopp — APPROVED** [review]
